hack/release.sh

```diff
@@ -14,5 +14,4 @@ echo "Building container image"
 # Using defaults, this pushes to:
 # quay.io/shipwright/shipwright-operator/github.com/shipwright-io/build/cmd/manager:latest
 KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS}" ko resolve -t "$TAG" -P -R -f deploy/ > release.yaml
-
-set +e
+KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS} -tags=pprof_enabled" ko resolve -t "$TAG-pprof" -P -R -f deploy/ > release-pprof.yaml
```

Comment:
I understand the rational, but I am not a fan of the tag `debug` as this might sound like one could connect a local debugger to the currently running operator to do live debugging.

Reply:
its a pattern adopted out there, see https://github.com/GoogleContainerTools/kaniko#debug-image . If is not debug, we need a more generic name, pprof is to specific.

Reply:
I see, but their `debug` image is actually an image to be used to debug their binary from inside the container. So the name makes sense. Anyway, if we feel that we should name it `debug` rather than `pprof`, I am ok with now.